test/support/example_multi_tenant.ex

```diff
@@ -0,0 +1,13 @@
+defmodule ExampleMultiTenant do
+  @moduledoc false
+  use Ash.Domain, otp_app: :ash_authentication
+
+  resources do
+    resource ExampleMultiTenant.Organisation
+    resource ExampleMultiTenant.User
+    resource ExampleMultiTenant.UserWithTokenRequired
+    resource ExampleMultiTenant.Token
+    resource ExampleMultiTenant.UserIdentity
+    resource ExampleMultiTenant.UserWithRegisterMagicLink
+  end
+end
```

Comment:
(I'm not an Ash maintainer, just another dev with Ash + multitenancy trying to help, so please prioritise any comments from the Ash team over mine) 

It feels odd to me to multi-tenant tokens? The first step for many apps is to authenticate, and _that_ informs which tenant is used from there on out. So the tenancy doesn't exist before the auth itself. 

I assume you're setting tenancy with some other method like a subdomain? 

(Possibly relevant discord link: https://discord.com/channels/711271361523351632/1253437793053577226/1341155047060537395) 

> I'm not sure why you'd bother making the token resource multi tenant tbh

Reply:
Multitenancy may also come from something like a subdomain or a header, so there is a case for it.

Reply:
I agreed with someone else's comment that its weird to add MT to a token, but also that it should be possible, so left it in :)